numpy/core/shape_base.py

```diff
@@ -179,12 +179,12 @@ def atleast_3d(*arys):
     True
 
     >>> for arr in np.atleast_3d([1, 2], [[1, 2]], [[[1, 2]]]):
-    ...     print(arr, arr.shape)
+    ...     print(arr, arr.shape) # doctest: +SKIP
     ...
     [[[1]
-      [2]]] (1, 2, 1)
+       [2]]] (1, 2, 1)
     [[[1]
-      [2]]] (1, 2, 1)
+       [2]]] (1, 2, 1)
```

Comment:
Something weird is going on with the spacing here. Why the doctest skip above?

Reply:
I fixed the spacing but we can't `print()` ndarrays with refguide at the moment -- so, in obvious cases where I can just use the repr I remove the `print()`, otherwise I just add a skip directive.